### PR TITLE
Improve Installing Instructions

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Improve installing instructions in [1].

[1] https://dev.ballerina.io/learn/installing-ballerina/
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/installing-ballerina.md [hunk 1]
```diff
@@ -17,44 +17,29 @@ redirect_from:
...
-:-----:|:-----:
-jBallerina version 1.2.0 or above|`ballerina dist update`
-jBallerina 1.2.0 but switched to a previous version|`ballerina dist pull jballerina-<LATEST-VERSION>`
-jBallerina version 1.1.0 or below|Use the [installers](#installing-ballerina-via-installers)
-
-
 ## Installing Ballerina via installers
 
-Ballerina installers support the below operating systems:
+Follow the below instructions to download and use the installers to install the latest Ballerina version. The installer will automatically uninstall the old Ballerina version if you have one already installed.
```
Comment (Samallama) [inline learn/installing-ballerina.md:30]: Shall we edit the first line to this: 

Follow the instructions below to install the latest Ballerina version using the installer.

The sentence here seems like a mouthful. WDYT?

File: learn/installing-ballerina.md [hunk 2]
```diff
@@ -17,44 +17,29 @@ redirect_from:
...
 
-- Windows 7 or later
-- Ubuntu Linux 
-- Red Hat Enterprise Linux
-- Cent OS
-- Mac OS 
+>**Info:** Ballerina installers support operating systems such as Windows, Ubuntu Linux, Red Hat Enterprise Linux, Maxc OS, and Cent OS. If you are using an unsupported operating system, [install via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file).
 
->**Info:** If you are using an unsupported operating system, [install via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file).
+### Installing on MacOS
```
Comment (Samallama) [inline learn/installing-ballerina.md:34]: Didn't we decide to make this macOS like the original site [1]?

[1] https://www.apple.com/macos/catalina/

File: learn/installing-ballerina.md [hunk 3]
```diff
@@ -17,44 +17,29 @@ redirect_from:
...
 
-Follow the below instructions to download and use the installers to install the latest Ballerina version. The installer will automatically uninstall the old Ballerina version if you have one already installed.
+[Download the package file](/downloads) and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution in the `/Library/Ballerina` directory.
 
-### Installing on Mac OS
+> **Info:** Alternatively, you can execute the below command to install Ballerina in MacOS using Homebrew.
```
Comment (Samallama) [inline learn/installing-ballerina.md:38]: The proper way to use below is after the noun so shall we edit it to:

Alternatively, you can execute the command below to install Ballerina in macOS using Homebrew.

If we have done this in other places, let's try to fix those as well

Comment (praneesha) [reply learn/installing-ballerina.md:38]: Fixed.

File: learn/installing-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/installing-ballerina.md:30]: Fixed.

File: learn/installing-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/installing-ballerina.md:34]: Fixed

File: learn/installing-ballerina.md [hunk 4]
```diff
@@ -70,28 +55,38 @@ The installer should put the `C:\Program Files\Ballerina\<ballerina-directory>\b
...
 
- - **For Linux or Mac OS:** set the PATH environment variable to point to the bin directory of the unzipped Ballerina distribution.
+ - **For Linux or MacOS:** set the PATH environment variable to point to the bin directory of the unzipped Ballerina distribution.
  - **For Windows:** add a new environment variable specifying the following values:
    - **Variable name:** PATH
    - **Variable value:** The location of the bin directory of the unzipped Ballerina distribution. For example, C:\Program Files\Ballerina\ballerina-<VERSION>\bin
+
+## Updating Ballerina
+
+If you already have a jBallerina version above 1.1.0 installed, you can use the update tool (by executing the below commands) to update to the latest jBallerina version by executing wither of the below commands.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply learn/installing-ballerina.md:83]: Fixed.

File: learn/installing-ballerina.md [hunk 5]
```diff
@@ -17,44 +17,29 @@ redirect_from:
...
 
-Ballerina installers support the below operating systems:
+Follow the instructions below to install the latest Ballerina version using the installer. The installer will automatically uninstall the old Ballerina version if you have one already installed.
 
-- Windows 7 or later
-- Ubuntu Linux 
-- Red Hat Enterprise Linux
-- Cent OS
-- Mac OS 
+>**Info:** Ballerina installers support operating systems such as Windows, Ubuntu Linux, Red Hat Enterprise Linux, Maxc OS, and Cent OS. If you are using an unsupported operating system, [install via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file).
```
Comment (Samallama) [inline learn/installing-ballerina.md:32]: should be CentOS and macOS, sorry missed this earlier

Comment (praneesha) [reply learn/installing-ballerina.md:32]: Fixed.

**Pull request merged**